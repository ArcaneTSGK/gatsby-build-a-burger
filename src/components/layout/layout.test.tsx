import * as React from 'react';
import { shallow } from 'enzyme';
import Layout from './layout';

describe('Layout Component', () => {
  let layout: any;

  beforeEach(() => {
    layout = shallow(<Layout children={<div></div>} />);
  });

  it('should not be null', () => {
    expect(layout).not.toBeNull();
  });

  it('Should render properly',() => {
    expect(layout).toMatchSnapshot();
  });
});