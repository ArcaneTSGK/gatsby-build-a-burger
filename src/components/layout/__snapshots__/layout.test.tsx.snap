// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component Should render properly 1`] = `
<Styled(div)>
  <ForwardRef(render)
    styles={[Function]}
  />
  <main>
    <div />
  </main>
</Styled(div)>
`;
